src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- Fixed a buggy TTL descriptor repair by removing it. Previously, upgrading from v22.2.x to 23.1.9 incorrectly removed [TTL storage parameters](https://www.cockroachlabs.com/docs/v23.1/row-level-ttl#ttl-storage-parameters) from tables (visible via [`SHOW CREATE TABLE  <ttl-table>`](https://www.cockroachlabs.com/docs/v23.1/show-create)) while attempting to repair table descriptors. This resulted in the node that attempts to run the TTL [job](https://www.cockroachlabs.com/docs/v23.1/show-jobs.html) crashing due to a panic caused by the missing TTL storage parameters. Clusters currently on v22.2.x should **not** be upgraded to v23.1.9 and should be upgraded directly to v23.1.10 or later. For more information, see [Technical Advisory 110363](../advisories/a110363.html). [#110500][#110500]
+- Fixed a bug causing performance regression when disabling `sql.metrics.statement_details.enabled` which caused execution stats to be collected for all queries instead of the default one percent. [#109881][#109881]
+- `cockroach debug pebble` commands now work correctly with encrypted stores which don't use the default `cockroach-data` path without having to also pass `--store`. [#110507][#110507]
+- The [**Database** pages](https://www.cockroachlabs.com/docs/cockroachcloud/databases-page) will no longer display `undefined` regions and outdated node information. [#110741][#110741]
+- Fixed a bug where `RESET (ttl_expire_after)` could incorrectly remove `ttl_expiration_expression`. [#110746][#110746]
+- Fixed a bug where an [`ALTER TABLE ... ADD CONSTRAINT CHECK ...`](https://www.cockroachlabs.com/docs/v23.1/alter-table#add-constraint) statement that utilized a user-defined function in the `CHECK` could cause a validation error. [#110720][#110720]
+- Fixed a bug where `CREATE INDEX` for [partial indexes](https://www.cockroachlabs.com/docs/v23.1/partial-indexes) could fail with `ERROR: duplicate key value violates unique constraint` if concurrent inserts happened simultaneously. [#110584][#110584]
+- Fixed a bug where a [`CREATE TABLE`](https://www.cockroachlabs.com/docs/v23.1/create-table) command with an `IDENTITY` column did not properly propagate the type of the column into the sequence. [#111014][#111014]
+- Fixed a bug where the `format_type` built-in did not honor `typemod` information for array types, leading to incorrect output. [#110940][#110940]
+- Fixed compaction behavior to prevent rapid sublevel growth when removing replicas from a store, such as during decommissioning. [#111141][#111141]
+- Fixed a bug that could occur when the "multiple active portals" execution mode (Preview) was enabled to evaluate queries such as lookup joins. The bug could result in an internal error like `unexpected 40960 leftover bytes` if the portal was not fully consumed. [#110666][#110666]
```

Comment:
```suggestion
- Fixed a bug that could occur when the "multiple active portals" execution mode (Preview) was enabled to evaluate queries such as lookup joins. The bug could result in an internal error like `unexpected 40960 leftover bytes` if the portal was not fully consumed. [#110666][#110666]
```
Link to somewhere for "multiple active portals"? Google didn't help me. Maybe there aren't docs.